samples/matter/lock/sample.yaml

```diff
@@ -66,7 +66,7 @@ tests:
   sample.matter.lock.smp_dfu.ffs:
     build_only: true
     extra_args: CONFIG_CHIP_DFU_OVER_BT_SMP=y CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE=y
-      CONFIG_CHIP_ROTATING_DEVICE_ID=y CONFIG_CHIP_DEVICE_TYPE=10
+      CONFIG_CHIP_ROTATING_DEVICE_ID=y CONFIG_CHIP_DEVICE_TYPE=10 CONFIG_ASSERT_NO_FILE_INFO=y
```

Comment:
Maybe that is something to be adjusted for debug + Wi-Fi configuration, change default argument for nrf7002dk_nrf5340_cpuapp platform.

Reply:
Discussed F2F